.gitignore

```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
 # Linux executables
 quickstart/cpp-linux/helloworld
 samples/cpp/windows/console/samples/sample
+*.dll
+*.exe
+samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/objd/amd64/speechtotext-wpf.exe.config
```

Comment:
>otext-wpf/objd/ [](start = 54, length = 15)

on top of the gitignrore file is a section to ignore build results, adding the objd there might be sufficient